stdlib/test/builtin/test_simd.mojo

```diff
@@ -154,22 +153,17 @@ def test_len():
     assert_equal(1, i4.__len__())
 
     alias UI64 = SIMD[DType.uint64, 16]
-    var i5 = UI64(10, 20, 30, 40)
-    assert_equal(16, i5.__len__())
+
     var i6 = UI64(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15)
     assert_equal(16, i6.__len__())
 
     @parameter
     if not has_neon():
         alias BF16 = SIMD[DType.bfloat16, 2]
-        var f1 = BF16(0.0)
```

Comment:
As mentioned in the other comment, here and below, I think the code intent was for specifying one element to mean "splat this value `N` times" where `N` is the simd width.